challenge_3/python/Loran425/src/main.py

```diff
@@ -0,0 +1,22 @@
+__author__ = "Loran425"
+
+
+def majority(data):
+    count = {}
+    output = None  # Have a value to return if there is no majority
+    for char in data:
+        # Dirty dirty dirty
+        try:
+            count[char] += 1
+        except KeyError:
+            count[char] = 1
+    for x in count.keys():
+        if count[x] >= len(count) // 2:
```

Comment:
You can do this check in the first for loop above and let you end the function faster.